firebase-ktx/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-platform-android")
+    id("org.jlleitschuh.gradle.ktlint")
+    id("maven-publish")
+    id("mirego.release").version("2.0")
+    id("mirego.publish").version("1.0")
+}
+
+repositories {
+    google()
+    mavenLocal()
+    mavenCentral()
+    maven("https://plugins.gradle.org/m2/")
+    maven("https://s3.amazonaws.com/mirego-maven/public")
+}
+
+group = "com.mirego.trikot.analytics"
+
+configurations.forEach { it.exclude("org.reactivestreams") }
+
+dependencies {
+    api(project(":analytics"))
+    api("com.mirego.trikot:streams:${project.extra["trikot_streams_version"]}")
+    api("com.mirego.trikot:trikotFoundation:${project.extra["trikot_foundation_version"]}")
+    implementation("com.google.firebase:firebase-analytics:18.0.2")
+    implementation("androidx.lifecycle:lifecycle-extensions:2.2.0")
+    implementation("androidx.lifecycle:lifecycle-reactivestreams-ktx:2.3.0")
+}
+
+android {
+    buildToolsVersion = "29.0.3"
```

Comment:
I don't think we need to specify a version of build tools anymore

Reply:
Removed